src/frontend/data/stores/useLivePanelState/index.spec.ts

```diff
@@ -0,0 +1,212 @@
+import { LivePanelDetail, useLivePanelState } from '.';
+
+describe('useLivePanelState()', () => {
+  it('init with default values', () => {
+    const state = useLivePanelState.getState();
+
+    expect(state.availableDetails).toEqual([]);
+    expect(state.currentDetail).toEqual(undefined);
+    expect(state.isPanelVisible).toBeFalsy();
+  });
+
+  it('change currentDetail using selectDetail', () => {
```

Comment:
Is it possible to organize the test so you have one test by each thing you want to test so it's easier to read and to maintain?